Source/Eto.WinForms/WinConversions.cs

```diff
@@ -403,7 +403,7 @@ public static MouseButtons ToEto(this swf.MouseButtons button)
 
 		public static Graphics ToEto(this sd.Graphics g)
 		{
-			return new Graphics(new GraphicsHandler(g));
+			return new Graphics(new GraphicsHandler(g, false));
```

Comment:
Hm, actually this is my mistake.. It won't dispose the graphics object unless the user code calls dispose as well.  Disregard my comments (;